mywebserver/notes/forms.py

```diff
@@ -0,0 +1,51 @@
+from django import forms
+from .models import WeekNote
+
+
+class WeekNoteForm(forms.ModelForm):
+    class Meta:
+        model = WeekNote
+        fields = ['day_of_week', 'note_title', 'note_description', 'assignee', 'email']
+
+    def clean_assignee(self):
+        assignee = self.cleaned_data.get('assignee')
+        if assignee and (len(assignee) < 5 or len(assignee) > 100):
```

Comment:
The length of the field is better to check with the field's validator.

clean_field method should be responsible for more complicated login in our case checking that the assignee has at least two words with the first capital letter in each word.